x-pack/plugins/data_usage/kibana.jsonc

```diff
@@ -12,5 +12,8 @@
     "requiredBundles": [
       "kibanaReact",
     ],
+    "extraPublicDirs": [
+      "common",
```

Comment:
Is there any way we can avoid using `extraPublicDirs`, it's been deprecated for a while and I believe the data_usage UI should be able to import directly from `common` if that's the intention?

Reply:
I added it to fix an error I was getting without it but it doesn't seem to be occurring anymore.  I've removed it.  Thanks!